code/datums/station_traits/negative_traits.dm

```diff
@@ -410,3 +410,257 @@
+	send_supply_pod_to_area(supply_pack_modsuits.generate(null), /area/station/science/robotics, /obj/structure/closet/supplypod/centcompod)
+
+	//Send a nebula shielding unit to engineering
+	var/datum/supply_pack/supply_pack_shielding = new /datum/supply_pack/engineering/rad_nebula_shielding_kit()
+	if(!send_supply_pod_to_area(supply_pack_shielding.generate(null), /area/station/engineering/main, /obj/structure/closet/supplypod/centcompod))
+		//if engineering isnt valid, just send it to the bridge
+		send_supply_pod_to_area(supply_pack_shielding.generate(null), /area/station/command/bridge, /obj/structure/closet/supplypod/centcompod)
+
+	//Disables radstorms, they don't really make sense since we already have the nebula causing storms
+	var/datum/round_event_control/modified_event = locate(/datum/round_event_control/radiation_storm) in SSevents.control
+	modified_event.weight = 0
+
+///They entered space? START BOMBING WITH RADS HAHAHAHA
+/datum/station_trait/nebula/hostile/radiation/proc/on_entered(area/space, atom/movable/enterer)
+	SIGNAL_HANDLER
+
+	if(!ismovable(enterer))
+		return
+
+	enterer.AddElement(/datum/element/radioactive, range = 0, minimum_exposure_time = NEBULA_RADIATION_MINIMUM_EXPOSURE_TIME)
+	//Don't actually make EVERY. SINGLE. THING. radioactive, just make them glow so people arent killed instantly
+	if(!SSradiation.can_irradiate_basic(enterer))
+		enterer.add_filter(GLOW_NEBULA, 2, list("type" = "drop_shadow", "color" = nebula_radglow, "size" = 2))
+
+///Called when an atom leaves space, so we can remove the radiation effect
+/datum/station_trait/nebula/hostile/radiation/proc/on_exited(area/space, atom/movable/exiter)
+	SIGNAL_HANDLER
+
+	exiter.RemoveElement(/datum/element/radioactive, range = 0, minimum_exposure_time = NEBULA_RADIATION_MINIMUM_EXPOSURE_TIME)
+	exiter.remove_filter(GLOW_NEBULA)
+
+/datum/station_trait/nebula/hostile/radiation/apply_nebula_effect(effect_strength = 0)
+	//big bombad now
+	if(effect_strength > 0)
+		if(!SSweather.get_weather_by_type(/datum/weather/rad_storm/nebula))
+			COOLDOWN_START(src, send_care_package_at, send_care_package_time)
+			SSweather.run_weather(/datum/weather/rad_storm/nebula)
+
+		//Send a care package to temporarily lift the storm!
+		if(COOLDOWN_FINISHED(src, send_care_package_at))
+			COOLDOWN_START(src, send_care_package_at, send_care_package_time)
+			var/obj/machinery/nebula_shielding/emergency/rad_shield = /obj/machinery/nebula_shielding/emergency/radiation
+
+			priority_announce(
+				{"Is everything okay there? We're getting high radiation readings from inside the station. \
+				We're sending an emergency shielding unit for now, it will last [initial(rad_shield.detonate_in) / (1 MINUTES)] minutes. \n\n\
+				Set up the nebula shielding. You can order construction kits at cargo if yours have been lost.
+				"}
+			)
+
+			addtimer(CALLBACK(src, PROC_REF(send_care_package)), 10 SECONDS)
+
+	//No storms, shielding is good!
+	else if(effect_strength <= 0)
```

Comment:
```suggestion
			addtimer(CALLBACK(src, PROC_REF(send_care_package)), 10 SECONDS)
			return

	//No storms, shielding is good!
```


Am I missing something or is this `else if` filling the same purpose that `else` would? 😅